### ISSUE TITLE
nextjs 15: ReactDebugCurrentFrame

### ISSUE THREAD

**Issue by m1guer** [pacocoursey/cmdk#324]
package.json:
```json
{
  "name": "....",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "NODE_OPTIONS='--inspect' next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint",
    "check-format": "biome check . --fix --unsafe",
    "check-lint": "biome lint . --fix",
    "format": "biome format . --fix",
    "prepare": "husky"
  },
  "dependencies": {
    "@hookform/resolvers": "^3.9.1",
    "@irsyadadl/paranoid": "^1.4.12",
    "@reverse-ui/react": "^1.0.11-alpha.3",
    "@tailwindcss/aspect-ratio": "^0.4.2",
    "@vercel/og": "^0.6.3",
    "@vercel/speed-insights": "^1.0.14",
    "axios": "^1.7.7",
    "class-variance-authority": "^0.7.0",
    "clsx": "^2.1.1",
    "cmdk": "^1.0.1",
    "date-fns": "^3.6.0",
    "embla-carousel-autoplay": "^8.3.1",
    "embla-carousel-react": "^8.3.1",
    "framer-motion": "^11.11.10",
    "input-otp": "^1.2.5",
    "justd-icons": "^1.7.1",
    "next": "^15.0.2",
    "next-themes": "^0.3.0",
    "next-view-transitions": "^0.3.2",
    "qs": "^6.13.0",
    "react": "^19.0.0-rc-603e6108-20241029",
    "react-aria": "^3.35.1",
    "react-aria-components": "^1.4.1",
    "react-dom": "^19.0.0-rc-603e6108-20241029",
    "react-hook-form": "^7.53.1",
    "react-number-format": "^5.4.2",
    "sharp": "^0.33.5",
    "sonner": "^1.5.0",
    "tailwind-merge": "^2.5.4",
    "tailwind-variants": "^0.2.1",
    "tailwindcss-animate": "^1.0.7",
    "tailwindcss-react-aria-components": "^1.1.6",
    "ws": "^8.18.0",
    "zod": "^3.23.8"
  },
  "devDependencies": {
    "@biomejs/biome": "1.8.3",
    "@eslint/js": "^9.13.0",
    "@fontsource/rosarivo": "^5.1.0",
    "@ianvs/prettier-plugin-sort-imports": "4.3.1",
    "@types/node": "^20.17.3",
    "@types/qs": "^6.9.16",
    "@types/react": "^18.3.12",
    "@types/react-dom": "^18.3.1",
    "@types/ws": "^8.5.12",
    "@typescript-eslint/eslint-plugin": "8.4.0",
    "@typescript-eslint/parser": "^8.12.2",
    "eslint": "8.57.0",
    "eslint-config-next": "^15.0.2",
    "eslint-config-prettier": "9.1.0",
    "eslint-import-resolver-typescript": "^3.6.3",
    "eslint-plugin-import": "2.30.0",
    "eslint-plugin-jsx-a11y": "^6.10.2",
    "eslint-plugin-prettier": "5.2.1",
    "eslint-plugin-unused-imports": "^4.1.4",
    "globals": "^15.11.0",
    "husky": "^9.1.6",
    "lint-staged": "15.2.10",
    "lucide-react": "^0.400.0",
    "postcss": "^8.4.47",
    "prettier": "3.3.3",
    "prettier-plugin-sort-json": "4.0.0",
    "prettier-plugin-tailwindcss": "^0.6.8",
    "tailwindcss": "^3.4.14",
    "typescript": "^5.6.3",
    "typescript-eslint": "^8.12.2"
  },
  "trustedDependencies": ["@biomejs/biome", "@vercel/speed-insights", "sharp"]
}
```

![Screenshot 2024-10-30 at 23 11 22](https://github.com/user-attachments/assets/403c6a50-52d6-4132-8b59-818f22785255)

**Comment by steve-marmalade** [comment]
Came here to report the same, please let me know if there's any info about my environment that would help to debug this :pray:

**Comment by lednhatkhanh** [comment]
Experiencing the same issue

**Comment by 0xj4m35** [comment]
Same with me

**Comment by rnkdsh** [comment]
+1

**Comment by parweb** [comment]
same same

**Comment by kjetil-fabnite** [comment]
same here

**Comment by ArnaudFeelbat** [comment]
Same using a fresh next js project initiate with next cli.

**Comment by roderik** [comment]
Downgrading to 1.0.0 makes it work again in Next 15 / React 19

**Comment by typeofweb** [comment]
1.0.1 still gives me that error. Should this issue be kept open until a fix is released?

**Comment by m1guer** [comment]
downgrade to 1.0

**Comment by irsyadadl** [comment]
yeah but you can keep it open.Bs the downgrade is the temporary sulution.

**Comment by pedro757** [comment]
+1

**Comment by pacocoursey** [comment]
Sorry, fixed in `1.0.2` now.
